### ISSUE TITLE
Add command to history

### ISSUE THREAD

**Issue by giggio** [nushell/nushell#11588]
### Related problem

I would like to be able to add a command to history, like in Bash where one can do `history -s foo`.

### Describe the solution you'd like

Add a subcommand to `history`. Like `history add` or `history insert`.

### Describe alternatives you've considered

None.

### Additional context and details

I'm trying to write the github-copilot-cli scripts for Nushell, and it has that. Specifically:

```bash
  copilot_what-the-shell () {
    TMPFILE=$(mktemp);
    trap 'rm -f $TMPFILE' EXIT;
    if /home/giggio/.n/bin/github-copilot-cli what-the-shell "$@" --shellout $TMPFILE; then
      if [ -e "$TMPFILE" ]; then
        FIXED_CMD=$(cat $TMPFILE);
        history -s $(history 1 | cut -d' ' -f4-); history -s "$FIXED_CMD";
        eval "$FIXED_CMD"
      else
        echo "Apologies! Extracting command failed"
      fi
    else
      return 1
    fi
  };
alias '??'='copilot_what-the-shell';
```

**Comment by fdncred** [comment]
What does `history -s foo` in bash, just insert `foo` into the history? What is the value of adding a command manually to the history? Also, if you're using the sqlite history, you could write a custom command to do that (I think) using `query db`.

**Comment by giggio** [comment]
Yes, [it inserts `foo` into history](https://ss64.com/bash/history.html). 
In this case, the github copilot cli is running a command it generated and then adding it to the history so the user can use it later.

**Comment by fdncred** [comment]
oh, that makes sense. I'm guessing we'd want this as a sub-command versus a parameter.

**Comment by giggio** [comment]
I just wrote it, I added it to my scripts, so I'm sharing here.
It seems quite dangerous to manipulate that table directly, I crashed Nushell several times and had to manually delete the rows to get it back to working condition. The `more_info` column is specially delicate, it seems.
